src/views/SpaceProposal.vue

```diff
@@ -267,12 +296,16 @@ onMounted(async () => {
         @clickVote="clickVote"
       />
       <BlockVotes
+        @loadVotes="loadMore(() => loadMoreVotes())"
```

Comment:
Is this wrapper function necessary?

```suggestion
        @loadVotes="loadMore(loadMoreVotes)"
```

Reply:
Yep works!